src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -2051,6 +2051,9 @@ protected override void BeginProcessing()
                 startInfo.WindowStyle = _windowstyle;
             }
 
+            string targetMessage = StringUtil.Format(ProcessResources.ProcessStartInfo, startInfo.FileName, startInfo.Arguments.Trim());
```

Comment:
The resource string id `ProcessStartInfo` should be changed to `StartProcessTarget`.

Reply:
@daxian-dbw Renamed the resource string as suggested.